spyder/plugins/ipythonconsole/utils/kernelspec.py

```diff
@@ -117,27 +117,35 @@ def argv(self):
                 self.set_conf('default', True, section='main_interpreter')
                 self.set_conf('custom', False, section='main_interpreter')
 
+        kernel_cmd = []
```

Comment:
```suggestion
        # Command used to start kernels
        kernel_cmd = []
```

Move comment from below.